x-pack/elastic-agent/pkg/agent/control/server/server.go

```diff
@@ -235,7 +235,13 @@ func (s *Server) ProcMeta(ctx context.Context, _ *proto.Empty) (*proto.ProcMetaR
 				endpoint = "npipe"
 			}
 
-			res, err := client.Get("http://" + endpoint + "/")
+			req, err := http.NewRequestWithContext(ctx, "GET", "http://"+endpoint+"/", nil)
```

Comment:
We can use http module constants here
```suggestion
			req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://"+endpoint+"/", nil)
```